chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
+	
+	for (let i=0, len=this._treebox.columns.count; i<len; i++) {
+		let column = this._treebox.columns.getColumnAt(i);
+		if (column.element.getAttribute('sortActive')) {
+			currentCol = column;
+			currentDir = column.element.getAttribute('sortDirection');
+		}
+		column.element.removeAttribute('sortActive');
+		column.element.removeAttribute('sortDirection');
+	}
+	
+	colId = Zotero.Prefs.get('itemTree.sortColumnId');
+	dir = Zotero.Prefs.get('itemTree.sortDirection');
+	// Restore previous sort setting
+	if (! this.collectionTreeRow.isFeed() && colId) {
+		col = this._treebox.columns.getNamedColumn(colId);
+		dir = Zotero.Prefs.get('itemTree.sortDirection');
+		Zotero.Prefs.set('itemTree.sortColumnId', null);
+		Zotero.Prefs.set('itemTree.sortDirection', null);
```

Comment:
I guess my question is more along the lines of "why are we unsetting this?" Seems like that should only be done if we're displaying items specifically in an unsorted state.